cookbook/enums.md

```diff
@@ -11,43 +11,43 @@ There is no direct support for enum's in JSON.  However, they are often encoded
 The above JSON object has a member, `"member0"`, that is an array of strings.
 In order to store the enum as a JSON string, an overload for `to_string` and `from_string` are required. `to_string` is required to return a `Container` where the `begin( )` iterator's `value_type` is `char, not necessarily a `std::string`.
 
-To see a working example using this code, refer to [cookbook_enums1_test.cpp](../tests/src/cookbook_enums1_test.cpp) 
+To see a working example using this code, refer to [cookbook_enums1_test.cpp](../tests/src/cookbook_enums1_test.cpp)
 ```c++
-enum class Colours : uint8_t { red, green, blue, black };
```

Comment:
Not all of us are American, not sure I am liking the spellchecking.  I can do that in IDE(apparently I haven't been in a couple of the files.)

Reply:
I'm not from the USA either, it just felt more natural in a programming context to not use British English.

Reply:
This change was implemented.